codegen/src/api/mod.rs

```diff
@@ -353,9 +387,33 @@ impl RuntimeGenerator {
                 where
                     T: ::subxt::Config,
                 {
+                    pub fn skip_pallet_validation(self) -> StorageApiUnchecked<'a, T> {
```

Comment:
Just a random thought; did you consider just having the unchecked functions alongside the checked ones i nthis same struct?

so youd end up with eg `balances()` and `balances_unchecked()` and so on for each pallet? it would skip the need for a separate struct and slightly condense the api from something like

```
api.tx().balances().skip_metadata_validation()...
```

to

```
api.tx().balances_unchecked()...
```

What do you reckon?